CMakeLists.txt

```diff
@@ -74,6 +74,16 @@ unset(CPACK_DEBIAN_PACKAGE_VERSION CACHE)
 
 include(IrodsCPackCommon)
 
+if (NOT CPACK_PACKAGING_INSTALL_PREFIX)
+  set(CPACK_PACKAGING_INSTALL_PREFIX "/" CACHE STRING "Package root path. \"/\" is correct for normal package builds.." FORCE)
+  message(STATUS "Setting unspecified CPACK_PACKAGING_INSTALL_PREFIX to '${CPACK_PACKAGING_INSTALL_PREFIX}'. This is the correct setting for normal builds.")
+endif()
```

Comment:
Sounds like we are trying to confirm whether this block is necessary in the other PRs, which would similarly affect this one, right?

Reply:
That is correct.